### DISCUSSION TITLE
How to get css word-break: break-word

### DISCUSSION THREAD

**Discussion by techopswhiz** [tailwindlabs/tailwindcss#11415] [Help] [answered]
[Looks](https://tailwindcss.com/docs/word-break) like tailwind has utilities for word-break, but I don't see the value `break-word`, and I wasn't able to achieve this functionality using any of the listed options. Is there an easy way to enable a utility for this, like by extending something in the config?

**Comment by wongjn** [comment, marked as answer]
You could use an [arbitrary property](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) like `[word-break:break-word]` but since you specifically asked for enabling it in configuration, you could add `word-break: break-word` as [a static utility](https://tailwindcss.com/docs/plugins#static-utilities) like:

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  plugins: [
    require('tailwindcss/plugin')(({ addUtilities }) => {
      addUtilities({
        '.break-word-legacy': {
          wordBreak: 'break-word',
        },
      });
    }),
  ],
};
```
See this working in [this Tailwind Play](https://play.tailwindcss.com/Elba6XKT7D?file=config) (open the *Generated CSS* panel).

**Reply by mybearworld** [reply]
It would be easier to just do it in the base CSS
```css
@layer components {
  .break-word-legacy {
    word-break: break-word;
  }
}
```

**Reply by wongjn** [reply]
Yes, although the OP specifically asked for a solution in config.

**Reply by mybearworld** [reply]
Right, sorry, I missed that.

**Reply by techopswhiz** [reply]
Legacy you say - why is this considered legacy? None of the default-supported options achieved the desired effect.

**Reply by wongjn** [reply]
[As per the CSS specication](https://drafts.csswg.org/css-text/#word-break-property):

> For compatibility with legacy content, the [`word-break`](https://drafts.csswg.org/css-text/#propdef-word-break) property also supports a deprecated `break-word` keyword. When specified, this has the same effect as `word-break: normal` and [`overflow-wrap: anywhere`](https://drafts.csswg.org/css-text/#propdef-overflow-wrap), regardless of the actual value of the `overflow-wrap` property.
